src/dmd/expression.d

```diff
@@ -4295,6 +4295,7 @@ extern (C++) final class SymOffExp : SymbolExp
  */
 extern (C++) final class VarExp : SymbolExp
 {
+    bool hasCheckedAttrs;
```

Comment:
It's a little naughty, but I don't know of any other solution, and it'd be a shame to hold up this feature on such a minor problem, so OK, let's do it. I do, however, suggest adding a comment explaining why this exists.

Reply:
Done